docs/user_guide/current_limitations.rst

```diff
@@ -0,0 +1,31 @@
+Current known limitations
+=========================
+
+This page lists all the known limitations of the ``tqec`` library.
+
+.. note::
+
+    We aim at making this page as exhaustive as possible, but it will never be 100% exhaustive.
```

Comment:
I added this page to list all the things we do not support currently in `tqec` so that we can refer to it whenever someone is asking about feature X or why his/her code is not working as expected.

I personally do not find the proposed clarification really clarifying.

Reply:
> I added this page to list all the things we do not support currently in tqec so that we can refer to it whenever someone is asking about feature X or why his/her code is not working as expected.

My concern is that this page would duplicate with issues on the repository that are feature requests. For example, under your justification, pretty much any of the issues labeled with `enhancement` can be listed on this page. 

I can see an argument for this being an unrealistic concern, so I don't really mind. However, for my personal understanding, can you clarify the following: if someone has desired feature(s), should they make an issue, or should they submit a PR that updates this file with a description of their desired feature(s)?